src/fonts/gonville_metrics.ts

```diff
@@ -179,12 +179,26 @@ export const GonvilleMetrics = {
       },
       flag32ndDown: {
         shiftX: -0.16,
+        shiftY: 9,
       },
       flag64thDown: {
         shiftX: -0.16,
+        shiftY: 13,
       },
       flag128thDown: {
         shiftX: -0.16,
+        shiftY: 22,
+      },
+      flag8thUp: {},
```

Comment:
I think you don't need these empty ones, because you can set a default in the lookup.

Reply:
Done